diff --git a/server/generator/constants.go b/server/generator/constants.go
index 401f12e6..49dcf659 100644
--- a/server/generator/constants.go
+++ b/server/generator/constants.go
@@ -19,7 +19,7 @@ func init() {
     BUILD_REF = "%s"
     BUILD_DATE = "%s"
 }
-`, strings.TrimSpace(b.String()), time.Now().Format("20060102"))
+`, strings.TrimSpace(b.String()), time.Unix(@build_date@, 0).Format("20060102"))

 	f, err := os.OpenFile("../common/constants_generated.go", os.O_CREATE|os.O_WRONLY, os.ModePerm)
 	if err != nil {
